its/projects/WebApplication/Foo.vb

```diff
@@ -0,0 +1 @@
+﻿Console.WriteLine("Not used")
```

Comment:
Since the syntax is not valid, let's just express the intention here
```suggestion
'This is just an unused file with *.vb extension to trigger the sensor
```